app/javascript/tapp/fetch.js

```diff
@@ -139,7 +139,7 @@ export const unlockAssignment = (applicant, assignment) => {
 
 // export offers from CHASS (locking the corresponding assignments)
```

Comment:
Hmm.  Maybe this is an opportunity to rename.  We aren't really exporting the offers from chass. We are really exporting them from tapp.  Do we need "chass" in the route?